### PR TITLE
Add release notes 2.12.0

### PR DESCRIPTION
**Pull request by serinamarie**
[Preview](https://github.com/PrefectHQ/prefect/blob/release-2.12.0/RELEASE-NOTES.md)

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,91 @@
...
+
+```python title="hello.py"
+from prefect import flow
+
+@flow
+def hello(name = "Marvin"):
+    print(f"Hello {name}!")
+
+if __name__ == "__main__":
+    hello.serve(name="hourly-greeting", interval=3600)
```
Comment (serinamarie) [inline RELEASE-NOTES.md:21]: @desertaxle Do we mind being verbose about what `name` and `interval` are here, if this is intended toward new users?

Comment (serinamarie) [reply RELEASE-NOTES.md:21]: At least that `name` is the name of a deployment

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,91 @@
...
 
+## Release 2.12.0
+
+### Introducing `Flow.serve()`
+
+We're excited to introduce a radically simple way to deploy flows. 
+
+The new `.serve()` method available on every flow allows you to take your existing flows and schedule or trigger runs via the Prefect UI and CLI. 
+
+This addition makes it easier than it's ever been to get started with Prefect:
```
Comment (serinamarie) [inline RELEASE-NOTES.md:11]: get started with Prefect
```suggestion
This addition makes it easier than it's ever been to get started with Prefect:
```
should this be "deploy flows with Prefect"?

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,91 @@
 # Prefect Release Notes
 
+## Release 2.12.0
+
+### Introducing `Flow.serve()`
+
+We're excited to introduce a radically simple way to deploy flows. 
+
+The new `.serve()` method available on every flow allows you to take your existing flows and schedule or trigger runs via the Prefect UI and CLI.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:9]: Should we actually detail what it means to "serve" your flows?

File: RELEASE-NOTES.md (hunk 1 above)
Comment (desertaxle) [reply RELEASE-NOTES.md:23]: Added a comment to the example script. Let me know if more clarity would be helpful.

File: RELEASE-NOTES.md (hunk 3 above)
Comment (desertaxle) [reply RELEASE-NOTES.md:9]: I tried to detail that with

> allows you to take your existing flows and schedule or trigger runs via the Prefect UI and CLI.

Let me know if I can make that clearer.

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,96 @@
...
+- https://github.com/PrefectHQ/prefect/pull/10574
+- https://github.com/PrefectHQ/prefect/pull/10585
+
+
+### Possible second highlight
+- Update Prefect UI with new color scheme for light and dark modes — https://github.com/PrefectHQ/prefect/pull/10546
+- update favicons — https://github.com/PrefectHQ/prefect/pull/10578
+- remove nav icons — https://github.com/PrefectHQ/prefect/pull/10584
+- bump deps and use sticky token — https://github.com/PrefectHQ/prefect/pull/10583
+- fix flow run graph bg — https://github.com/PrefectHQ/prefect/pull/10588
```
Comment (serinamarie) [inline RELEASE-NOTES.md:54]: @brandonreid Can you add the highlight here?

Comment (serinamarie) [inline RELEASE-NOTES.md:54]:
```suggestion
See the following pull requests for implementation details;
    - https://github.com/PrefectHQ/prefect/pull/10546
    - https://github.com/PrefectHQ/prefect/pull/10578
    - https://github.com/PrefectHQ/prefect/pull/10584
    - https://github.com/PrefectHQ/prefect/pull/10583
    - https://github.com/PrefectHQ/prefect/pull/10588
```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,102 @@
...
+If you're excited to start serving your flows, check out our newly updated [quickstart](https://docs.prefect.io/latest/getting-started/quickstart/) and [tutorial](https://docs.prefect.io/latest/tutorial/) to get started.
+
+See the following pull requests for details:
+- https://github.com/PrefectHQ/prefect/pull/10534
+- https://github.com/PrefectHQ/prefect/pull/10549
+- https://github.com/PrefectHQ/prefect/pull/10574
+- https://github.com/PrefectHQ/prefect/pull/10585
+
+
+### An updated Prefect UI
```
Comment (billpalombi) [inline RELEASE-NOTES.md:49]:
```suggestion
### A fresh look and feel
```

**Review by billpalombi — COMMENTED** [review]
Minor suggestions throughout. Looks great!

File: RELEASE-NOTES.md (hunk 3 above)
Comment (serinamarie) [reply RELEASE-NOTES.md:9]: I think it could be but I'll defer to @billpalombi

Comment (billpalombi) [reply RELEASE-NOTES.md:9]: Yeah I think thats great

**Review by billpalombi — APPROVED** [review]
Looks great!

**Pull request merged**